tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/palm_detection.py

```diff
@@ -0,0 +1,255 @@
+"""
+Copyright (c) 2018-2022 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+
+from ..adapters import Adapter
+from ..config import StringField, NumberField, BoolField, ListField
+from ..representation import DetectionPrediction
+
+
+class PalmDetectionAdapter(Adapter):
+    __provider__ = 'palm_detection'
+
+    @classmethod
+    def parameters(cls):
+        params = super().parameters()
+        params.update({
+            'scores_out': StringField(description='scores output'),
+            'boxes_out': StringField(description='boxes output'),
+            'num_layers': NumberField(
```

Comment:
very common name, difficult to understand for what it is need, maybe such name will be better
```suggestion
            'num_anchor_layers': NumberField(
```

please also align description with documentation

Reply:
fixed